### ISSUE TITLE
environment/evironment div classname: fixing the typo breaks the installation page

### ISSUE THREAD

**Issue by szarnyasg** [duckdb/duckdb-web#1246]
Investigate #1245, cc @krlmlr

**Comment by szarnyasg** [comment]
So there are two similarly named CSS classes
* `.evironment` in`body.installation`
* `.install .environment` in `body.landing`

The related HTML looks as follows:

```html
<div class="evironment select">
	...
	<ul class="environment">
		...
	</ul>
	<label class="onlymobile">
		<select name="environment" class="environmentselect onlymobile">
			...
		</select>
	</label>
</div>
```

**Comment by szarnyasg** [comment]
Fixed by #1334
